apps/E2E/src/ActivityIndicator/specs/ActivityIndicator.spec.ios.ts

```diff
@@ -1,22 +1,12 @@
-import NavigateAppPage from '../../common/NavigateAppPage';
 import ActivityIndicatorPageObject from '../pages/ActivityIndicatorPageObject';
-import { PAGE_TIMEOUT, BOOT_APP_TIMEOUT } from '../../common/consts';
 
 // Before testing begins, allow up to 60 seconds for app to open
 describe('Activity Indicator Testing Initialization', function () {
   it('Wait for app load', async () => {
-    await NavigateAppPage.waitForPageDisplayed(BOOT_APP_TIMEOUT);
-    await expect(await NavigateAppPage.isPageLoaded()).toBeTruthy(NavigateAppPage.ERRORMESSAGE_APPLOAD);
+    await ActivityIndicatorPageObject.waitForInitialPageToDisplay();
   });
 
   it('Click and navigate to Activity Indicator test page', async () => {
-    await ActivityIndicatorPageObject.mobileScrollToComponentButton();
-    await ActivityIndicatorPageObject.waitForButtonDisplayed(PAGE_TIMEOUT);
-
-    /* Click on component button to navigate to test page */
-    await NavigateAppPage.clickAndGoToActivityIndicatorPage();
-    await ActivityIndicatorPageObject.waitForPageDisplayed(PAGE_TIMEOUT);
-
-    await expect(await ActivityIndicatorPageObject.isPageLoaded()).toBeTruthy(ActivityIndicatorPageObject.ERRORMESSAGE_PAGELOAD);
+    await ActivityIndicatorPageObject.navigateToPageAndLoadTests();
```

Comment:
Does this cause an issue since there's no `expect` call here?

Reply:
From testing, a `waitFor` timing out causes a test to fail because it will throw an error. The spec still acts the same, but putting an `expect` call here would be good for sanity.

Reply:
The expects have been re-added